cpg_utils/hail_batch.py

```diff
@@ -40,9 +40,9 @@ def init_batch(**kwargs):
         return  # already initialised
     dataset = get_config()['workflow']['dataset']
     kwargs.setdefault('token', os.environ.get('HAIL_TOKEN'))
-    kwargs.setdefault('default_reference', genome_build())
     asyncio.get_event_loop().run_until_complete(
         hl.init_batch(
+            default_reference=genome_build(),
             billing_project=get_config()['hail']['billing_project'],
             remote_tmpdir=remote_tmpdir(f'cpg-{dataset}-hail'),
             **kwargs,
```

Comment:
What happens if `default_reference` is also specified in the `kwargs`? Does the latter take precedence? Or are you trying to forbid that? (This seems to be the only functional change, right?)

Reply:
I'm reverting to preserve the original behaviour, which in this case which cause a TypeError (`init_batch() got multiple values for keyword argument 'default_reference'`. To be clear, I want to forbid this introduced behaviour of allowing the override of the default reference.